docs/mobile-apps/automated-testing/appium/real-devices.md

```diff
@@ -2,190 +2,209 @@
+In order to install your app on the devices you select, you must identify the location from which your mobile app can be accessed. You can specify a Sauce Labs Application Storage ID or filename, or a remote location to which Sauce Labs has access.
 
-:::caution Limited Support for *.aab Files
-At this time, \*.aab files are only supported for Android real device testing.
-:::
+```js title=App Storage Example
+caps.setCapability("app","storage:filename=mapp.zip");
+```
 
-### Application Storage and Data Center Endpoints
+```js title=Remote App Example
+caps.setCapability("app","https://github.com/test-apps/ios-app.zip");
+```
 
-When uploading your app for testing with real devices, you must identify the [Data Center](/basics/data-center-endpoints) where the devices you are testing are located by specifying the applicable API URL for that data center.
 
-Examples using the EU and US endpoints:
+### Setting the `appiumVersion`
 
-<Tabs
-  defaultValue="US Data Center"
-  values={[
-    {label: 'US Data Center', value: 'US Data Center'},
-    {label: 'EU Data Center', value: 'EU Data Center'},
-  ]}>
+If you omit the `appiumVersion` in your test configuration, your test runs with our default Appium version, which is typically the version that supports the broadest number of device combinations, but may not be the latest version. We recommend that you specify the newest Appium version that supports the specific devices and operating systems you plan to use for your test, so you can leverage the most up-to-date features and patches.
 
-<TabItem value="US Data Center">
+```js
+caps.setCapability("appiumVersion", "1.5.3");
+```
 
-US Data Center (macOS/Linux)
+#### **Checking the Appium Version for Your Test**
+1. Log into Sauce Labs.
+2. Go to **Test Details**.
+3. Find and select the test that you ran using Appium.
+4. Click the **Metadata** tab.
+5. Look for the **Logs** row and select **Appium Log**. The first line should indicate the Appium version. For example, `2019-05-05T17:45:07.541Z - info: Welcome to Appium v1.10.1`.
 
-```bash
-curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
---request POST 'https://api.us-west-1.saucelabs.com/v1/storage/upload' \
---form 'payload=@"g16K4P8IX/iOS.RealDevice.SauceLabs.Mobile.Sample.app.2.7.1.ipa"' \
---form 'name="iOS.RealDevice.SauceLabs.Mobile.Sample.app.2.7.1.ipa"'
-```
+### Excluding the `browserName`
 
-US Data Center (Windows)
+When testing a native mobile app, no browser is accessed, so if you are re-using the capabilities from your mobile or desktop browser tests, you can omit the `browserName` capability or or set the value as an empty string. This is an important exclusion because if values are set for _both_ `app` and `browserName`, Sauce Labs defaults to the `browserName`. Similarly, if neither capability is specified, Sauce Labs defaults to the `browserName` settings of Safari for iOS and Chrome for Android.
```

Comment:
> ...or set the value as an empty string...

Please remove this, we need to _educate_ customers to use proper caps, this is a bad practice, so please remove it